bin/perseo

```diff
@@ -105,7 +105,8 @@ function loadConfiguration() {
         'PERSEO_AUTHENTICATION_HOST',
         'PERSEO_AUTHENTICATION_PORT',
         'PERSEO_AUTHENTICATION_USER',
-        'PERSEO_AUTHENTICATION_PASSWORD'
+        'PERSEO_AUTHENTICATION_PASSWORD',
+        'PERSEO_AUTHENTICATION_SERVICE'
```

Comment:
Should this new PERSEO_AUTHENTICATION_SERVICE be included in documentation?

Reply:
It's related with previous issue https://github.com/telefonicaid/perseo-fe/issues/638, it's and improvement